src/sitemap/sitemaps.py

```diff
@@ -98,7 +101,10 @@ def _urls(self, page, protocol, domain):
         urls = super()._urls(page, splitted_url.scheme, splitted_url.hostname)
         for url in urls:
             # Add information about alternative languages
-            url["alternates"] = url["item"].sitemap_alternates
+            try:
+                url["alternates"] = url["item"].sitemap_alternates
+            except AttributeError:
+                logger.debug("%s has no attribute sitemap_alternates", url["item"])
```

Comment:
```suggestion
            url["alternates"] = self.sitemap_alternates(url["item"])
```
The sitemap alternates indicate the same content but in a different language. Thus, it makes sense to include them also for offers (sorry that I forgot about this in the issue). See my suggestion below how you could include them.